### ISSUE TITLE
`enforce-canonical-classes` misses some issues

### ISSUE THREAD

**Issue by nelsonlaidev** [schoero/eslint-plugin-readable-tailwind#313]
### Description

Sorry to bother you. I may have misconfigured something. Could you help me out?

<img width="2308" height="1720" alt="Image" src="https://github.com/user-attachments/assets/205c4115-4130-4338-abab-4c42d78cd483" />

### Flavor

HTML

### Code Input

```html
<div class="mt-[16px] [display:flex] size-[22px] max-w-[90px]">
  Hello
</div>
```

### Expected Behavior

```html
<div class="mt-4 flex size-5.5 max-w-22.5">
    Hello
</div>
```

### Actual Behavior

```html
<div class="mt-[16px] flex size-[22px] max-w-[90px]">
    Hello
</div>
```

### Reproduction URL

https://github.com/nelsonlaidev/better-tailwindcss-test

### ESLint Log

```shell
/Users/nelson/Developer/playground/better-tailwindcss-test/index.html
  11:31  warning  The class: "[display:flex]" can be simplified to "flex"  better-tailwindcss/enforce-canonical-classes

✖ 1 problem (0 errors, 1 warning)
  0 errors and 1 warning potentially fixable with the `--fix` option.
```

### ESLint Config

```javascript
import eslintParserHTML from "@html-eslint/parser";
import eslintPluginBetterTailwindcss from "eslint-plugin-better-tailwindcss";
import { defineConfig } from "eslint/config";

export default defineConfig({
  extends: [eslintPluginBetterTailwindcss.configs.recommended],

  settings: {
    "better-tailwindcss": {
      entryPoint: "global.css",
    },
  },

  rules: {
    "better-tailwindcss/enforce-shorthand-classes": "off",
    "better-tailwindcss/enforce-consistent-important-position": "off",
    "better-tailwindcss/enforce-consistent-variable-syntax": "off",
  },

  files: ["**/*.html"],

  languageOptions: {
    parser: eslintParserHTML,
  },
});
```

### Versions

```json
{
  "@html-eslint/parser": "^0.54.0",
  "eslint": "^9.39.2",
  "eslint-plugin-better-tailwindcss": "^4.0.2",
  "tailwindcss": "^4.1.18"
}
```

**Comment by schoero** [comment]
You need to configure the [`rootFontSize`](https://github.com/schoero/eslint-plugin-better-tailwindcss/blob/main/docs/rules/enforce-canonical-classes.md#rootfontsize)

```jsonc
  //...
  rules: {
    "better-tailwindcss/enforce-canonical-classes": ["warn", { "rootFontSize": 16 }],
  },
  //...
```
